src/api/hooks/useGetANS.ts

```diff
@@ -22,8 +22,6 @@ export const knownAddresses: Record<string, string> = {
     "Legacy Token (0x3)",
   "0x0000000000000000000000000000000000000000000000000000000000000004":
     "Digital Assets (0x4)",
-  "0x000000000000000000000000000000000000000000000000000000000000000a":
```

Comment:
why the removal here

Reply:
I think it showed up funny in search.  I could go either way here